Readability.js

```diff
@@ -506,6 +506,8 @@ Readability.prototype = {
         }
 
         while (p.lastChild && this._isWhitespace(p.lastChild)) p.removeChild(p.lastChild);
+
+        if (p.parentNode.tagName === "P") this._setNodeTag(p.parentNode, "DIV");
```

Comment:
So my understanding is that this code is meant to deal with `<br>` separated paragraphs by turning them into actual paragraphs. If we end up doing this *inside* a `<p>`, wouldn't the most semantically correct thing be to make the new `<p>` a sibling of the existing `<p>` ?

In other words, why did you end up going for making these `<div>`s ? Maybe there's a reason not to do what I'm suggesting that I'm missing (maybe just complexity) ?

Reply:
I did it this way mostly because it was the simplest - the end result is the same, as a new paragraph gets created for the content before the `br` chain by

https://github.com/mozilla/readability/blob/d4b842c82a86c09d1f5c9f8ea8781692ba3cd147/Readability.js#L794-L799

the only difference being that all those new paragraphs get grouped together in a `div` (`<p>foo<br><br>bar</p>` -> `<div><p>foo</p><p>bar</p></div>`)

Reply:
OK, fair enough. Let's do this then. :-)